src/app/components/PromptInput/PromptInputContainer.tsx

```diff
@@ -70,7 +71,11 @@ export const PromptInputContainer = ({
         .prompt-input-inner {
           position: relative;
           margin: 0 auto;
-          width: ${isMobile ? 'calc(100% - 40px)' : '700px'};
+          width: ${isMobile
+            ? 'calc(100% - 40px)'
+            : isTablet
+            ? '700px'
```

Comment:
I guess this fixes this?

https://github.com/athrael-soju/Iridium-AI/pull/97#issuecomment-1789476780

Reply:
Yeah